tsconfig.json

```diff
@@ -1,18 +1,42 @@
 {
   "compilerOptions": {
-    "baseUrl": "src",
-    "rootDir": "./",
-    "outDir": "build",
-    "target": "esnext",
+    // Basic Options
+    "target": "ES2017",
     "module": "commonjs",
-    "moduleResolution": "node",
+    "declaration": true,
     "sourceMap": true,
-    "removeComments": true,
+
+    // TODO remove lib once BigInt is polyfilled
```

Comment:
I'm not sure this TODO makes sense. What are you using `lib` here for?

Reply:
By default the Lib value is set to only allow in the codebase what the target `ES2017` allows. I overwrite this in the lib section so we can use BigInt `ESNext`. Once BigInt is polyfilled we should remove the lib configuration value and rely on the defaults.

Reply:
Do we still need this now that we are polyfilling?

Reply:
The polyfill is in a different PR https://github.com/xpring-eng/Xpring-JS/pull/150

Reply:
Should that other PR remove `lib` in `tsconfig`?

Reply:
That PR was prior to the TSConfig changes but now that it is merged I can remove that for this PR :)